### DISCUSSION TITLE
QQL with a new column having constant value

### DISCUSSION THREAD

**Discussion by ShivaniSK** [finos/TimeBase-CE#241] [Q&A] [answered]
Hi @alex-karpovich ,
Is there a way to get a column with constant value in response?
E.g
select 'ASK' as orderType, ask_price, ask_size from orderbook;

the expected output should be

orderType   ask_price  ask_size
  ASK              12.1         11
  ASK               33.3         12

Regards,
Shivani

**Comment by alex-karpovich** [comment]
yes, it will work

**Reply by ShivaniSK** [reply]
Hi @alex-karpovich ,
select 'ASK' as orderType, ask_price, ask_size from orderbook; -- this works 
However the below query does not work as expected

`(select 'ask' as order_type, ask_price as price from order_book_depth) UNION (select 'bid' as order_type, bid_price as price from order_book_depth)`

The output does not show the column "order_type" with BID and ASK constants.

**Reply by alex-karpovich** [reply, marked as answer]
please try the following:

`
(select ('ask' if true) as order_type, ask_price as price from order_book_depth) UNION (select ('bid' if true) as order_type, bid_price as price from order_book_depth)
`

using ```'ask' as order_type``` - creates static column, which is not supported yet for unions

**Reply by alex-karpovich** [reply]
@ShivaniSK is it working for you?

**Reply by ShivaniSK** [reply]
hello @alex-karpovich , yes this works fine :) thanks
